tools/test/examples/examples_lib.py

```diff
@@ -391,7 +391,7 @@ def compile_repos(config, toolchains, targets, profile, examples):
                                                                 valid_choices(example['toolchains'], toolchains),
                                                                 example['features']):
                     print("Compiling %s for %s, %s" % (name, target, toolchain))
-                    build_command = ["mbed-cli", "compile", "-t", toolchain, "-m", target, "-v"]
+                    build_command = ["python", "-m", "mbed", "compile", "-t", toolchain, "-m", target] + (['-v'] if verbose else [])
```

Comment:
This change is not explained in the commit message or in the Pull Request description. Could you please explain why you added `python -m` to the beginning of this command?

Reply:
Removed.